app/src/main/java/net/squanchy/support/widget/SpeakerView.java

```diff
@@ -90,15 +91,25 @@ private void updateSpeakerPhotos(List<Speaker> speakers, OnSpeakerClickListener
         }
 
         for (Speaker speaker : speakers) {
-            ImageView photoView = recycleOrInflatePhotoView(photoViews);
-            speakerPhotoContainer.addView(photoView);
             if (speaker.photoUrl().isPresent()) {
-                photoView.setOnClickListener(v -> listener.onSpeakerClicked(speaker));
+                ImageView photoView = recycleOrInflatePhotoView(photoViews);
+                speakerPhotoContainer.addView(photoView);
+                setClickListenerOrNotClickable(photoView, listener, speaker);
                 loadSpeakerPhoto(photoView, speaker.photoUrl().get(), imageLoader);
             }
         }
     }
 
+    private void setClickListenerOrNotClickable(ImageView photoView, Optional<OnSpeakerClickListener> listener, Speaker speaker) {
+        if (listener.isPresent()) {
+            photoView.setOnClickListener(v -> listener.get().onSpeakerClicked(speaker));
+            photoView.setClickable(true);
+        } else {
+            photoView.setOnClickListener(null);
+            photoView.setClickable(false);
```

Comment:
In which, order is _extremely_ important. `setOnClickListener` makes the view clickable even if you pass it a `null`.